crates/bevy_asset/src/path.rs

```diff
@@ -723,6 +698,29 @@ impl FromReflect for AssetPath<'static> {
     }
 }
 
+pub(crate) trait NormalizedPath {
+    fn normalized(&self) -> PathBuf;
```

Comment:
Doc strings would be nice here.

Reply:
I've added a one-liner description - let me know if you think anything else should be added to it